zoo-css-challenge/style.css

```diff
@@ -85,45 +90,71 @@ footer {
   border-radius: 50%;
   display: inline-block;
   position: fixed;
-  bottom: 20px;
+  /* bottom: 20px; */
```

Comment:
```suggestion
```
We should delete commented code instead of committing them, which is what most software teams do